File: translations/codenet/gpt_commented/C2J/granite/Code/s325320702.java

import java.util.*;
import java.io.*;

class s325320702 {
    public static void main(String[] args) throws Exception {
        // Read input values for l, r, and d
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        long l = Long.parseLong(br.readLine());
        long r = Long.parseLong(br.readLine());
        long d = Long.parseLong(br.readLine());

        // Count the multiples of d in the range [l, r]
        long ans = 0;
        for (long i = l; i <= r; i++) {
            if (i % d == 0) ans++;
        }

        // Output the result
        System.out.println(ans);
    }
}

File: translations/codenet/uncommented/C2P/deepseek/Code/s325320702.py
import sys

def main():
    l, r, d = map(int, sys.stdin.readline().split())
    ans = 0
    for i in range(l, r + 1):
        if i % d == 0:
            ans += 1
    print(ans)

if __name__ == "__main__":
    main()
